src/main/java/com/example/gamelist/GameController.java

```diff
@@ -24,13 +27,23 @@ public  List<Game> gamelist(){
     }
 
 
+
     @PostMapping("/gamelist")
     public ResponseEntity<Response> create(@RequestBody GameForm form){
         URI url = UriComponentsBuilder.fromUriString("http://localhost:8080")
                 .path("/gamelist/id")
                 .build()
                 .toUri();
-        return ResponseEntity.created(url).body(new Response("name successfully created"));
+        return ResponseEntity.created(url).body(new Response("gamelist successfully created"));
+    }
+
+
+
+    @PatchMapping("/gamelist/{id}")
+    public ResponseEntity<UpdateResponse> UpdateGamelist(@PathVariable int id, @RequestBody GamelistService.UpdateRequest updateRequest){
+        Gamelists.update(id,updateRequest.getName());
```

Comment:
更新処理を呼び出すためにGamelistServiceを使いましょうか。
```suggestion
        gamelistService.更新するためのメソッド(id, updateRequest.getName());
```
`更新するためのメソッド`の部分はお茶さんが実装して GamelistService の更新のために作ったメソッド名に置き換えて下さい。

Reply:
![スクリーンショット 2023-12-21 23 09 52](https://github.com/otyacazyu/gamelist/assets/143002834/66951fd8-72db-4091-8424-ab8f00d21337)
236813ba684f84fe35bdffa0c0f346ee6df570c5

更新の為のメソッドがわからなくなってしまい、UpdateRequestクラスを作成して見ました。
合ってないような気がします・・・ご助言いただけたらと思います

Reply:
いったんこちらもresolveでよいです。